runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -245,17 +291,20 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
-	 * @brief Create an array of elements for a construct FFI type
+	 * @brief Create an array of elements for a construct FFI type.
 	 *
-	 * @param layout[in] A pointer to a c string describing the types of the struct elements
-	 * @return An array of ffi_type* which are the elements of the struct
+	 * @param layout[in/out] A pointer to a C string describing the types of the struct elements,
+	 * which points to the end of the struct layout string upon return
+	 * @return An array of ffi_type * which are the elements of the struct
 	 */
-	ffi_type**
-	getStructFFITypeElements(char **layout);
+	ffi_type **
+	getStructFFITypeElements(const char **layout);
 
 	/**
-	 * @brief Create a struct FFI type
-	 * @param layout[in] A pointer to a c string describing the types of the struct elements.
+	 * @brief Create a struct FFI type.
+	 *
+	 * @param layout[in/out] A pointer to a C string describing the types of the struct elements,
+	 * which points to the end of the struct layout string upon return.
```

Comment:
Suggest this should go before line 349 (the `@return` tag).

Reply:
Fixed.